extensions/amp-form/0.1/amp-form.js

```diff
@@ -585,6 +648,11 @@ export class AmpForm {
    * @private
    */
   handleNonXhrGet_(varSubsFields) {
+    if (this.ssrTemplateHelper_.isSupported()) {
+      this.assertSsrTemplate_('Non-XHR requests are not supported for viewers '
+          + 'with renderTemplate capabilities');
+      return;
+    }
```

Comment:
```js
this.assertSsrTemplate_(true, 'Non-XHR GETs not supported.');
```

Return shouldn't be necessary since a JS error is thrown, but let's double-check this on a "dist" build.